app/views/add-reduction.html

```diff
@@ -17,78 +17,91 @@
     {% set fromAction='/offender-manager/{{ linkId }}/add-reduction' %}    
   {% endif %}
 
-  <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">{{ headerText }}</header>
+  {% include "includes/validation-error-messages.html" %}
 
-  {%if failureText %}
-  <p class="message message--failure">{{ failureText }}</p>
-  {% endif %}
+  <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">{{ headerText }}</header>
 
   <p class="lede">{{ reductionText }}<span class="bold">{{ title }}</span></p>
   {% set first = true %}
   <form action="{{ formAction }}" method=POST id="reductionForm">
     <input type="hidden" name="reductionId" value="{{reduction.id}}"/>
     <div class="form-group" id="reduction_reason_selection">
```

Comment:
The IDs for the form elements should probably be updated to the value referred to by the error message link e.g. `reasonForReductionId`

Reply:
This is the case for each view.

Reply:
As long as name matches we're OK with the id being different

Reply:
Ah, I didn't realise that. Thanks for the clarification.